### DISCUSSION TITLE
multi module projects

### DISCUSSION THREAD

**Discussion by elwgomes** [nvim-java/nvim-java#292] [Q&A]
i'm working at a multi-module project and something is bothering me and i guess there is a solution but idk 

so, how can i create new module from nvim-java?
every time i need to create a new module i need to open an ide

all answers are welcome hehe

**Comment by bavo** [comment]
Can't you create the modules in gradle of maven manually? 
Or are you using some different build tool.

**Reply by elwgomes** [reply]
i can, but it  take so many time to create manually... other code editor are faster so...
